tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from app import create_app, db
+
+@pytest.fixture
+def app():
+  app = create_app()
+  app.config.from_object('config.TestingConfig')
+  with app.app_context():   
+    db.create_all()
+    yield app
+    db.session.close()
+    db.drop_all()
```

Comment:
This fixture allows us to access an instance of the app in all of our tests and takes care of setup and teardown of the db.